src/gexf/common/parser.js

```diff
@@ -143,6 +143,8 @@ function collectAttributes(model, defaults, element) {
     valueElement = valueElements[i];
     id = valueElement.getAttribute('id') || valueElement.getAttribute('for');
 
+    if(!model[id]) throw new Error('Found attribut \'' + id + '\' which is not defined');
```

Comment:
The error message should be `graphology-gexf/parser: Found undeclared attribute "blabla".`